packages/fxa-admin-server/src/config.ts

```diff
@@ -153,6 +153,12 @@ const conf = convict({
       },
     },
   },
+  ipHmacKey: {
+    doc: 'A secret to hash IP addresses for security history events',
+    format: String,
+    default: 'changeme',
+    env: 'IP_HMAC_KEY',
+  },
```

Comment:
I know `changeme` looks a little wild, but I wrote it to try to match what we have here: https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/config/index.ts#L1483